translations/translation_fr.xml

```diff
@@ -153,12 +153,13 @@
 
 		<text name="CP_vehicle_courseGeneratorSetting_isClockwise_title"								text="Sens des fourrières" />
 		<text name="CP_vehicle_courseGeneratorSetting_isClockwise_tooltip"								text="Sens horaire ou antihoraire pour les fourrières." />
-		
-
 		<text name="CP_vehicle_courseGeneratorSetting_isClockwise_clockwise" 							text="Sens horaire" />
 		<text name="CP_vehicle_courseGeneratorSetting_isClockwise_counterclockwise" 					text="Sens antihoraire" />
 		
 		<text name="CP_vehicle_courseGeneratorSetting_centerMode_title"									text="Gestion des rangs" />
+		
```

Comment:
Can you pls remove those extra lines so the setting_centerMode translation stays together?

Reply:
Yes of course you are right.